### DISCUSSION TITLE
Qfield photo location

### DISCUSSION THREAD

**Discussion by FatimaCaballeroIC** [opengisch/QField#4009] [Q&A]
Hello everyone. I have a qgis project, which I export via wire with qfieldsync. In one of my tables I have a field that is photos, which is set to attachment. The problem is that no matter how much I configure the path and the name of the file, relative to the project, it saves the images under the name DCIM-timestamp.jpg and in the directory /Android/data/ch.opengis.qfield/Imported Projects/ files/projectname/DCIM.
I want to know if it is possible to save the photos taken by the tablet in a different location.

**Comment by lucienicolier** [comment]
Hi @FatimaCaballeroIC,

Did you have a look at the doc? 
Maybe you will find your answer in here, [Configurable picture path](https://docs.qfield.org/how-to/pictures/#configurable-picture-path).

Please tell me if it helps.

Best
